### ISSUE TITLE
Icons next to project links lack aria-hidden

### ISSUE THREAD

**Issue by seanfagan** [seanfagan/seanfagan.github.io#12]
All the pretty fontawesome icons should have the attribute `aria-hidden="true"` (assuming they're superfluous, which they should be), but most do not.
